dashboard/server/src/main/java/build/bazel/dashboard/github/issuestatus/GithubIssueStatusService.java

```diff
@@ -124,70 +125,64 @@ static Status newStatus(GithubRepo repo, GithubIssue.Data data) {
     Instant updatedAt = data.getUpdatedAt();
 
     switch (status) {
-      case TO_BE_REVIEWED:
-      case MORE_DATA_NEEDED:
-      case REVIEWED:
+      case TO_BE_REVIEWED, MORE_DATA_NEEDED, REVIEWED -> {
```

Comment:
I think switch statements allow you to be even more concise now, e.g. write here:

```
case TO_BE_REVIEWED, MORE_DATA_NEEDED, REVIEWED -> updatedAt.plus(7, DAYS);
```

Similar for the `default` case below

Reply:
Done.